### DISCUSSION TITLE
notebooks/fds/2

### DISCUSSION THREAD

**Discussion by giscus** [NoughtQ/notebook#8] [Announcements]
# notebooks/fds/2

https://note.noughtq.top/notebooks/fds/2

**Comment by Marquez404** [comment]
写的好好，4.3多了个,逗号

**Comment by NoughtQ** [comment]
感谢指出 :smile:。其实我感觉这里写的有些啰嗦了，这一块不是重点内容，只是为后续算法分析打下基础
